src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {Store} from '@ngrx/store';
+import {ACTION} from '../../../models/hearings.enum';
+import {HearingsService} from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+import {RequestHearingPageFlow} from '../request-hearing.page.flow';
+
+@Component({
+  selector: 'exui-hearing-facilities',
+  templateUrl: './hearing-facilities.component.html',
+})
+export class HearingFacilitiesComponent extends RequestHearingPageFlow implements OnInit, OnDestroy {
+
+  constructor(protected readonly hearingStore: Store<fromHearingStore.State>,
+              protected readonly hearingsService: HearingsService) {
+    super(hearingStore, hearingsService);
+  }
+
+  public ngOnInit(): void {
+  }
+
+  public executeAction(action: ACTION): void {
+    if (this.isFormValid()) {
+      super.navigateAction(action);
+    }
+  }
+
+  public isFormValid(): boolean {
+    // TODO verify if form group is valid
```

Comment:
unnecessary comment

Reply:
I deliberately add this // TODO comment, as when implement the component it needs to follow the same pattern for form validation under the page flow. This // TODO should be removed after the validation is added but not now.